ompi/mpi/man/man3/MPI_Bcast.md

```diff
@@ -0,0 +1,141 @@
+
+# NAME
+==================================
```

Comment:
A few style notes:

1. No need for blank lines at the beginning of the file
1. No need for the `====` lines (both here and later in the file) -- using `# NAME` is sufficient to make a level-1 heading with the word `NAME`.  The `====` didn't seem to render as anything in Markdown, either (check out how your file rendered on the Github web interface).